xml/admin_monitoring_alerting.xml

```diff
@@ -126,8 +126,24 @@
       </para>
     </listitem>
   </itemizedlist>
-  <para>For more information on supplying your own SSL certificates, see <xref linkend="cert-sign-CA"/>
-    or for creating your own, see <xref linkend="self-sign-certificates"/>.
+  <para>
+    The &dashboard; embeds the &grafana; dashboards via iframe HTML
```

Comment:
Is the method (iframe) here really important? Or is it an implementation detail of the HTML? I would vote to remove it.

Reply:
@LenzGr

Reply:
> Is the method (iframe) here really important? Or is it an implementation detail of the HTML? I would vote to remove it.

it is quite important as that is the reason we need the SSl certificate. iframed pages behave like a stand-alone pages and nowadays browsers just refuse to view iframed sites thru plain http (at least i believe)